packages/platforms/ingrasys/x86-64/x86-64-ingrasys-s9100/modules/builds/eeprom.c

```diff
@@ -0,0 +1,237 @@
+/*
```

Comment:
Please don't duplicate kernel drivers in your platform. 
Remove this copy and configure CONFIG_EEPROM_LEGACY as a module in the 3.16 kernel config.